docs/src/main/sphinx/connector/hive.rst

```diff
@@ -623,6 +623,8 @@ Property Name                              Description
        ``KERBEROS``. Default is ``NONE``.
    * - ``hive.metastore.thrift.impersonation.enabled``
      - Enable Hive metastore end user impersonation.
```

Comment:
`Read spark generated statistics in hive connector` -> `Fallback to spark generated statistics if Trino statistics are absent`

Reply:
I'm not a native English speaker, but both sentences look very similar and have very similar meaning (use statistics from Apache spark if hive statistics are not available vs are absent).